templates/pipelines/common-app-ci.yaml

```diff
@@ -48,7 +51,10 @@ parameters:
   
 variables:
 - group: NodeJSAppBuild
-
+- name: IsMainBranch
+  value: ${{ eq(variables['Build.SourceBranch'], 'refs/heads/main') }}
+- name: IsPrBuild
+  value: ${{ eq(variables['Build.Reason'], 'PullRequest') }}
 stages:         
   - stage: Application_CI
```

Comment:
can we have more friendly display names.. even just Application Ci (without underscores) in the ADO UI